package.json

```
@@ -39,21 +39,21 @@
     "release": "semantic-release"
   },
   "dependencies": {
-    "node-fetch": "3.3.2"
+    "isomorphic-unfetch": "^4.0.2"
```

```suggestion
    "isomorphic-unfetch": "4.0.2"
```

Remove carrtor here too